admin/bin/qserv-version.sh

```diff
@@ -20,10 +20,53 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 
-
-# Return Qserv version number
-# Used to determine if Qserv run dir is compliant with Qserv version
+# Return Qserv version or release number
 
 # @author  Fabrice Jammes, IN2P3
-QSERV_VERSION="2016_07"
-echo "${QSERV_VERSION}"
+
+QSERV_RELEASE="2016_07"
+
+set -e
+
+usage() {
+  cat << EOD
+
+Usage: `basename $0` [options]
+
+  Available options:
+    -h          this message
+    -R          print release number
+
+  Print Qserv (eups-based) version number,
+  or Qserv release on which this version is based
```

Comment:
'version number' is computed by eups and depends on git tag, 'release' is set by hand each month.